src/main/java/com/dropbox/core/android/AuthActivity.java

```diff
@@ -387,6 +442,7 @@ protected void onCreate(Bundle savedInstanceState) {
     protected void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
         outState.putString(SIS_KEY_AUTH_STATE_NONCE, mAuthStateNonce);
+        outState.putSerializable(SIS_KEY_PKCE_MANAGER, mPKCEManager);
```

Comment:
I forget - will this automatically serialize and deserialize the fields of the `DbxPKCEManager` class?  IIRC this approach has compat issues if you change the class on an update and this code tries to deserialize an old one.  Better to be explicit and serialize any simple types, and then on deserialize call a constructor with them